src/containers/Authorization/SignInForm.tsx

```diff
@@ -50,6 +57,9 @@ export const SignInForm: React.FC<ISignInForm> = (props) => {
 
   return (
     <FormWrapper text={t('AUTH.TITLE_LOGIN_FORM')}>
+      <DropDownWrapper>
+        <DropDown onChange={useChangeLanguage()} options={languages}/>
```

Comment:
const onLanguageChange =  useChangeLanguage()

and then <DropDown onChange={onLanguageChange}.......